Library/Formula/truncate.rb

```diff
@@ -0,0 +1,39 @@
+class Truncate < Formula
+  desc "truncates a file to a given size"
+  homepage "http://www.vanheusden.com/truncate"
+  url "http://www.vanheusden.com/truncate/truncate-0.8.tgz"
+  sha256 "165914d643351324d69096427d3ce5d3ab13aed3f7c990a71ba3bd0c6f278f00"
+
+  patch :DATA # remove missing file; rename syscalls for OSX which are natively 64bit
```

Comment:
Ok it is on github now: https://github.com/flok99/truncate

On Sat, Jan 9, 2016 at 9:55 PM, Folkert van Heusden <mail@vanheusden.com>
wrote:

> Please wait 30 minutes.
>
> On Sat, Jan 9, 2016 at 12:01 PM, galaxy001 <notifications@github.com>
> wrote:
>
>> In Library/Formula/truncate.rb
>> <https://github.com/Homebrew/homebrew/pull/47883#discussion_r49263497>:
>>
>> > @@ -0,0 +1,39 @@
>> > +class Truncate < Formula
>> > +  desc "truncates a file to a given size"
>> > +  homepage "http://www.vanheusden.com/truncate"
>> > +  url "http://www.vanheusden.com/truncate/truncate-0.8.tgz"
>> > +  sha256 "165914d643351324d69096427d3ce5d3ab13aed3f7c990a71ba3bd0c6f278f00"
>> > +
>> > +  patch :DATA # remove missing file; rename syscalls for OSX which are natively 64bit
>>
>> The developer is @flok99 <https://github.com/flok99> on Github. But
>> truncate is not in his public repos.
>>
>> https://github.com/search?q=user%3Aflok99+truncate&type=Code
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/Homebrew/homebrew/pull/47883/files#r49263497>.
>>
>
>
>
> --
> www.vanheusden.com  www.slimwinnen.nl | www.smartwinning.info
>



-- 
www.vanheusden.com  www.slimwinnen.nl | www.smartwinning.info